debian/not-installed

```diff
@@ -0,0 +1 @@
+etc/opt/chrome/native-messaging-hosts/eu.webeid.json
```

Comment:
Note for self: this will be installed with postinst script: https://github.com/web-eid/web-eid-app/blob/main/debian/web-eid-chrome.postinst